Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -175,7 +173,10 @@ open class CallViewModel: ObservableObject {
     private var callEventsSubscriptionTask: Task<Void, Never>?
     private var participantsSortComparators = defaultSortPreset
     private let callEventsHandler = CallEventsHandler()
-    private var localCallSettingsChange = false
+
+    /// The variable is `true` if CallSettings have been set on the CallViewModel instance (directly or indirectly).
+    /// The variable will be reset to `false` when `leaveCall` will be invoked.
+    private(set) var localCallSettingsChange = false
```

Comment:
Shouldn't this be configurable though? In meeting apps, I would expect the settings to be remembered between call sessions.

Reply:
So, what i have in my mind is that customers so far are going around this problem by providing their own callSettings whenever they join or create a call. So meeting apps can do that. Basically i'm disabling the propagation as it seems to be unexpected.